src/cloudsc_cuda/cloudsc/cloudsc_driver.cu

```diff
@@ -467,21 +467,25 @@ void cloudsc_driver(int numthreads, int numcols, int nproma) {
     zfrac = (double)igpc / (double)numcols;
     if (tloc > 0.0) {
       zmflops = 1.0e-06 * zfrac * zhpm * ((double)numcols / 100.) / tloc;
+      zthrput = (double)numcols/tloc;
     } else {
       zmflops = 0.;
+      zthrput = 0.0;
     }
-    printf(" %10d%10d%10d%10d%10d %4d : %10d%10d @ core#\n",
-	   numthreads, numcols, igpc, icalls, nproma, t, (int)(tloc * 1000.), (int)zmflops);
+    printf(" %10d%10d%10d%10d%10d %4d : %10d%10d%10d @ core#\n",
+ 	   numthreads, numcols, igpc, icalls, nproma, t, (int)(tloc * 1000.), (int)zmflops, (int)zthrput);
   }
   double tdiff = t2 - t1;
   zfrac = 1.0;
   if (tdiff > 0.0) {
     zmflops = 1.0e-06 * zfrac * zhpm * ((double)numcols / 100.) / tdiff;
+    zthrput = (double)numcols/tdiff;
   } else {
     zmflops = 0.0;
+    zthrput = 0.0;
   }
-  printf(" %10d%10d%10d%10d%10d %4d : %10d%10d TOTAL\n",
-	 numthreads, numcols, numcols, nblocks, nproma, -1, (int)(tdiff * 1000.), (int)zmflops);
+  printf(" %10d%10d%10d%10d%10d %4d: %10d%10d%10d TOTAL\n",
```

Comment:
Ther's now no white space before the colon `:`. Just wondering if that screws up alignment of the table?